dask/bytes/core.py

```diff
@@ -106,9 +111,13 @@ def read_bytes(urlpath, delimiter=None, not_zero=False, blocksize=2**27,
         token = tokenize(fs_token, delimiter, path, fs.ukey(path),
                          compression, offset)
         keys = ['read-block-%s-%s' % (o, token) for o in offset]
-        out.append([delayed_read(OpenFile(fs, path, compression=compression),
-                                 o, l, delimiter, dask_key_name=key)
-                    for o, key, l in zip(offset, keys, length)])
+        values = [delayed_read(OpenFile(fs, path, compression=compression),
+                               o, l, delimiter, dask_key_name=key)
+                  for o, key, l in zip(offset, keys, length)]
+        if include_path:
+            out.append((path, values))
+        else:
+            out.append(values)
```

Comment:
cc @martindurant .  What are your thoughts on this change to dask.bytes?

Reply:
I'd rather build up a separate list of paths, and return that as well, rather than returning a list of tuples of (path, list).